### PR TITLE
Comments out morphs

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Removes the morph ghost role spawn. Its weight is 0 so I dunno how it spawns at all.

## Changelog

:cl:
Del: Morph ghost event. Its weight is 0 and yet somehow it still spawns.
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Failing to properly pass the test because the file that I unincluded is not included. Aka it worked well. Advice?

**Comment by projectkepler-ru** [comment]
convert to draft PR, I'll see about locally testing this

**Comment by Cyprex** [comment]
Don't remove this file, that'll just cause errors. This spawns because Skyrat's custom event_chaos_system doesn't take into account weights and such- Commenting out the event in event_chaos_system.dm might work though.

File: tgstation.dme [hunk 1]
```diff
@@ -3193,7 +3193,7 @@
 #include "code\modules\events\ghost_role\alien_infestation.dm"
 #include "code\modules\events\ghost_role\blob.dm"
 #include "code\modules\events\ghost_role\fugitive_event.dm"
-#include "code\modules\events\ghost_role\morph_event.dm"
+//#include "code\modules\events\ghost_role\morph_event.dm" BUBBER EDIT
```
Comment (NullDagaf) [inline tgstation.dme:3196]: This is a terrible way of doing this.

**Pull request closed without merging**